Source/API/EbApi.h

```diff
@@ -38,6 +38,33 @@ extern "C" {
 #define EB_NON_REF_PICTURE   4
 #define EB_INVALID_PICTURE   0xFF
 
+#define EB_SEGMENT_BLOCK_SIZE 64
+    typedef struct
+    {
+        uint8_t ovFlags;
+        int8_t  qpOv;
+        int8_t  deblockOv;
+        int8_t  filterOv;
```

Comment:
I don't see where deblockOv and filterOv are initialized or assigned. Should be in the Loop Filter process?

Reply:
Fixed by implementation of 2nd idea ("SegmentOvFile")